.github/workflows/ci.yml

```diff
@@ -0,0 +1,101 @@
+name: CI
+
+on:
+  schedule:
+    - cron: 0 2 * * 1-5
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  tests:
+    name: Tests PHP ${{ matrix.php }} ${{ matrix.dependency }} (Symfony ${{ matrix.symfony }})
+    runs-on: ubuntu-latest
+    container:
+      image: php:${{ matrix.php }}
+      options: >-
+        --tmpfs /tmp:exec
+    strategy:
+      # https://github.community/t5/GitHub-Actions/GitHub-Actions-Matrix-options-dont-work-as-documented/m-p/29558
+      matrix:
+        php:
+          - '5.5'
+          - '5.6'
+          - '7.0'
+          - '7.1'
+          - '7.2'
+          - '7.3'
+          - '7.4'
+          - '8.0.0RC2'
+        symfony:
+          - '3.4.*'
+          - '4.4.*'
+          - '5.0.*'
+        include:
+          - php: '7.1'
+            coverage: true
+          - php: '7.1'
+            dependency: 'dunglas/symfony-lock:^3'
```

Comment:
We could stop using this package and use flex with `SYMFONY_REQUIRE: 3.4.*` instead.